src/constants/index.ts

```diff
@@ -213,4 +213,5 @@ export const WYRE_API_URL = 'https://api.sendwyre.com'
 export const WYRE_QUOTE_API_ENDPOINT = '/v3/orders/quote/partner'
 export const WYRE_RESERVE_API_ENDPOINT = '/v3/orders/reserve'
 export const WYRE_CALLBACK_URL = 'https://app.pangolin.exchange/'
-export const ENABLE_STABLESWAP: boolean = process.env.REACT_APP_ENABLE_STABLESWAP === 'true'
```

Comment:
You can add `REACT_APP_ENABLE_STABLESWAP="true"` to your `.env.local`

Reply:
I know I was too lazy 😅🥲